AdobeUi/view/adminhtml/web/js/components/grid/masonry.js

```diff
@@ -152,6 +151,28 @@ define([
             }.bind(this));
         },
 
+        /**
+         * Waits for container.
+         */
+        waitForContainer: function (callback) {
+            if (this.imports.rows.length === 0 || typeof this.container === "undefined") {
```

Comment:
```suggestion
            if (this.rows.length === 0 || typeof this.container === "undefined") {
```
Imports defaults configuratioin is actually setting the `rows` property